q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,219 @@
+declaration is made, they are not reinstated automatically, but may run in the
+Special Election to fill the vacancy.
+
+Otherwise, Steering Council members may only be removed before the end of
+their term by a No Confidence Vote by the Core Team.
+
+## Voting
+
+### Vote Administration
+
+Every vote requires a Vote Administrator who manages communication, collection
+of secret ballots, and all other necessary activities to complete the voting
+process.
+
+Unless otherwise specified, the Steering Council selects the Vote
+Administrator.
+
+A Vote Administrator must not be a member of the Steering Council nor a
+candidate or subject of the vote.  A Vote Administrator may be a member of the
+Core Team and, if so, may cast a vote while also serving as administrator.  If
+the Vote Administrator becomes a candidate during an election vote, they will
+appoint a non-candidate replacement.
+
+If the entire Steering Council is vacant or is the subject of a No Confidence
+Vote, then the Core Team will select a Vote Administrator by consensus.  If
+consensus cannot be reached within one week, the President of The Perl
+Foundation will select a Vote Administrator.
+
+### Term and Special Elections
+
+Term and Special Election are ranked-choice votes to construct an ordered list
+of candidates to fill vacancies in the Steering Council.
+
+Both Term and Special Elections occur in two stages:
+
+1. Candidates advertise their interest in serving. Candidates must be
+   nominated by a Core Team member. Self-nominations are allowed.  Nominated
+   candidates may share a statement about their candidacy with the Core Team.
+
+2. Active Core Team Members vote by ranking all candidates.  Voting is
+   performed anonymously.  After voting is complete, candidates are ranked
+   using the Condorcet Minimax method.  If a tie occurs, it may be resolved by
+   mutual agreement among the tied candidates, or else the tie will be
+   resolved through random selection by the Vote Administrator.
+
+For a Term Election, each phase will last two weeks.  At the end of the second
+phase, the top five ranked candidates are elected as the new Steering Council.
+
+For a Special Election, each phase will last one week.  At the end of the
+second phase, vacancies are filled from the ordered list of candidates until
+no vacancies remain.
+
+If a Steering Council candidate is not already a member of the Core Team, they
+are added to the Core Team if elected to the council.
+
+### Membership Change, Amendment, and No Confidence Votes
+
+Membership Change, Amendment, and No Confidence Votes are super-majority votes
+by the Core Team.
+
+They occur in two steps:
+
+1. The Vote Administrator describes proposal being voted upon.
+
+2. Active Core Team members vote in favor or against the proposal.  Voting is
+   performed anonymously.
+
+If 2/3 or more of the votes cast are in favor of the proposal, the proposal
+succeeds and goes into effect.
```

Comment:
I appreciate the intent to avoid stalling things, but the zero quorum feels problematic. The rules above imply that Core Team includes at least the 5 Steering Committee members, who are expected to be active; I think it would not be unreasonable to say that if fewer than 3 votes are cast the vote is invalid and must be re-run or withdrawn.

Reply:
I hear your concern, but I feel pretty strongly about this point.  To even get to this point either 3 steering council members are in favor of something (for membership/amendment) or two core team members are in favor of getting rid of at least 1 steering council member, so there are 3 people with a vested interest either way.

And, again, if we're at a point where fewer than 3 people bother to show up for votes, the system has failed anyway, so I'm not interested in special casing for that situation.

Reply:
Yes, the system has failed; but it is not failing safe.

Reply:
The alternative is deadlock.  E.g. everyone resigns but one person.  That person has no quorum to add people to achieve quorum.  I'm OK with the system not failing safe if the number of people interested in Perl governance becomes  that small.

Reply:
I agree with not designing for the failure case.